### DISCUSSION TITLE
Can I overwrite username regex?

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2752] [Q&A] [answered]
Can I overwrite this `^[\w][\w\.]*$` username regex?

**Comment by ganigeorgiev** [comment, marked as answer]
This will change sometime in the future (_see https://github.com/pocketbase/pocketbase/discussions/2425#discussioncomment-5818276_), but at the moment it is not possible to change the usernames allowed pattern.
